ddtrace/internal/processor/trace.py

```diff
@@ -141,6 +141,9 @@ def on_span_finish(self, span):
                 else:
                     finished = trace_spans
 
+                if self._partial_flush_enabled and trace.num_finished >= self._partial_flush_min_spans:
+                    log.debug("Partially flushing %d spans for trace %d", len(trace.spans), span.trace_id)
+
                 trace.num_finished -= len(finished)
```

Comment:
since we are doing `len(finished)` twice, might be worth doing it in a variable?

```python
num_finished = len(finished)

if partial_flushing:
    log.debug(num_finished)

trace.num_finished -= num_finished
```